applications/dashboard/views/entry/registerbasic.php

```diff
@@ -24,7 +25,7 @@
             <li>
                 <?php
                 echo $this->Form->label('Username', 'Name');
-                echo $this->Form->textBox('Name', array('Wrap' => true, 'autocorrect' => 'off', 'autocapitalize' => 'off'));
+                echo $this->Form->textBox('Name', array('autocorrect' => 'off', 'autocapitalize' => 'off', 'Wrap' => TRUE));
```

Comment:
There used to be 2 views, registerbasic and registercaptcha.
Due to the "refactor" done by Mark in the before times, Captcha-less registration was not supported but both views still existed. 

Various overrides existed in the code to ensure that even though `Basic` was the only option provided in the UI, `Captcha` is what ended up in the config and caused `registercaptcha` to be used. 

I just properly consolidated this logic. Thereafter, I fixed these views by deleting `registerbasic` and renaming `registercaptcha` to `registerbasic`. This is what was already in `registercaptcha`.